consensus/merge/src/main/java/org/hyperledger/besu/consensus/merge/MergeContext.java

```diff
@@ -59,7 +63,7 @@ long addNewForkchoiceMessageListener(
 
   Optional<Block> retrieveBlockById(final PayloadIdentifier payloadId);
 
-  void fireNewForkchoiceMessageEvent(
+  void fireNewUnverifiedForkchoiceMessageEvent(
```

Comment:
This has been renamed, because we fire this message even in the case that we deem the forkchoice state as invalid. It is needed in order to make pivot selection, for the initial sync after the merge.